ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -515,14 +513,24 @@ objects:
         }
 
         function run-upgrade-tests() {
+          # Set up bastion host to ensure tests can actually use the proxy          
+          setup_ssh_bastion
+          export KUBE_SSH_BASTION=${BASTION_HOST}
           openshift-tests run-upgrade "${TEST_SUITE}" --to-image "${RELEASE_IMAGE_LATEST}" \
             --provider "${TEST_PROVIDER:-}" -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit
+          echo "Removing ssh-bastion"
```

Comment:
Iam explicitly calling the functions that deploy bastion host pod in these `tests*` functions. To be clear, we're still going to fail some tests in if we start relying on the wrong environment variable

Reply:
I brought up an issue here: https://github.com/eparis/ssh-bastion/pull/9 that the bastion being deployed in the `openshift-ssh-bastion` namespace is causing optional e2e tests to fail. I ended up closing the PR, but if we're going to deploy a bastion for every e2e test, we should revisit this issue. 

Here is the e2e test that will fail: https://github.com/openshift/origin/blob/master/test/extended/operators/images.go#L81